src/main/java/com/google/cloud/resourcemanager/OrgPolicyInfo.java

```diff
@@ -0,0 +1,397 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.resourcemanager;
+
+import com.google.api.services.cloudresourcemanager.model.OrgPolicy;
+import com.google.api.services.cloudresourcemanager.model.RestoreDefault;
+import com.google.common.base.Function;
+import com.google.common.base.MoreObjects;
+import java.io.Serializable;
+import java.util.List;
+import java.util.Objects;
+
+/** A Google Cloud Resource Manager organization policy metadata object. */
+public class OrgPolicyInfo implements Serializable {
+  static final Function<OrgPolicy, OrgPolicyInfo> FROM_PB_FUNCTION =
+      new Function<com.google.api.services.cloudresourcemanager.model.OrgPolicy, OrgPolicyInfo>() {
+        @Override
+        public OrgPolicyInfo apply(
+            com.google.api.services.cloudresourcemanager.model.OrgPolicy pb) {
+          return OrgPolicyInfo.fromPb(pb);
+        }
+      };
+  static final Function<OrgPolicyInfo, com.google.api.services.cloudresourcemanager.model.OrgPolicy>
+      TO_PB_FUNCTION =
+          new Function<
+              OrgPolicyInfo, com.google.api.services.cloudresourcemanager.model.OrgPolicy>() {
+            @Override
+            public com.google.api.services.cloudresourcemanager.model.OrgPolicy apply(
+                OrgPolicyInfo orgPolicyInfo) {
+              return orgPolicyInfo.toPb();
+            }
+          };
+  private static final long serialVersionUID = 9148970963697734236L;
+
+  private BooleanPolicy booleanPolicy;
+  private String constraint;
+  private String etag;
+  private ListPolicy listPolicy;
+  private RestoreDefault restoreDefault;
+  private String updateTime;
+  private Integer version;
+
+  /** Used in policy_type to specify how booleanPolicy will behave at this resource. */
+  public static class BooleanPolicy implements Serializable {
+    private static final long serialVersionUID = -2133042982786959351L;
+    private final Boolean enforce;
+
+    BooleanPolicy(Boolean enforce) {
+      this.enforce = enforce;
```

Comment:
This is nullable? Why?

Reply:
Because you can pass boolean value here while creating Instance of it. i.e  `OrgPolicyInfo.BoolPolicy booleanPolicy= new OrgPolicyInfo.BoolPolicy(true);`
for more details you can see [here](https://cloud.google.com/resource-manager/reference/rest/v1/Policy#booleanpolicy)